examples/lambdarank/README.md

```diff
@@ -21,6 +21,10 @@ You should finish training first.
 
 Run the following command in this folder:
 
+```bash
+../../lightgbm config=predict.conf
 ```
-"./lightgbm" config=predict.conf
-```
+
+Data Format
+-----------
```

Comment:
Separate heading from text with one blank line, like it's done for `Training` section.